command/sidechain/terminate-ban/terminate_ban.go

```diff
@@ -0,0 +1,118 @@
+package terminateban
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/0xPolygon/polygon-edge/command/helper"
+	"github.com/0xPolygon/polygon-edge/command/polybftsecrets"
+	"github.com/0xPolygon/polygon-edge/command/sidechain"
+	sidechainHelper "github.com/0xPolygon/polygon-edge/command/sidechain"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/contractsapi"
+	"github.com/0xPolygon/polygon-edge/contracts"
+	"github.com/0xPolygon/polygon-edge/txrelayer"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/spf13/cobra"
+	"github.com/umbracle/ethgo"
+)
+
+var (
+	params terminateBanParams
+)
+
+func GetCommand() *cobra.Command {
+	terminateBanCmd := &cobra.Command{
+		Use:     "terminate-ban",
+		Short:   "Terminates the temporary ban for validator",
+		PreRunE: runPreRun,
+		RunE:    runCommand,
+	}
+
+	helper.RegisterJSONRPCFlag(terminateBanCmd)
+	setFlags(terminateBanCmd)
+
+	return terminateBanCmd
+}
+
+func setFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(
+		&params.accountDir,
+		polybftsecrets.AccountDirFlag,
+		"",
+		polybftsecrets.AccountDirFlagDesc,
+	)
+
+	cmd.Flags().StringVar(
+		&params.accountConfig,
+		polybftsecrets.AccountConfigFlag,
+		"",
+		polybftsecrets.AccountConfigFlagDesc,
+	)
+
+	cmd.Flags().BoolVar(
+		&params.insecureLocalStore,
+		sidechain.InsecureLocalStoreFlag,
+		false,
+		"a flag to indicate if the secrets used are encrypted. If set to true, the secrets are stored in plain text.",
+	)
+
+	cmd.MarkFlagsMutuallyExclusive(polybftsecrets.AccountDirFlag, polybftsecrets.AccountConfigFlag)
+}
+
+func runPreRun(cmd *cobra.Command, _ []string) error {
+	params.jsonRPC = helper.GetJSONRPCAddress(cmd)
+
+	return params.validateFlags()
+}
+
+func runCommand(cmd *cobra.Command, _ []string) error {
+	outputter := command.InitializeOutputter(cmd)
+	defer outputter.WriteOutput()
+
+	validatorAccount, err := sidechainHelper.GetAccount(
+		params.accountDir,
+		params.accountConfig,
+		params.insecureLocalStore,
+	)
+	if err != nil {
+		return err
+	}
+
+	txRelayer, err := txrelayer.NewTxRelayer(txrelayer.WithIPAddress(params.jsonRPC),
+		txrelayer.WithReceiptTimeout(150*time.Millisecond))
+	if err != nil {
+		return err
+	}
+
+	encoded, err := contractsapi.HydraChain.Abi.Methods["terminateBanProcedure"].Encode(
+		[]interface{}{},
+	)
+	if err != nil {
+		return err
+	}
+
+	txn := &ethgo.Transaction{
+		From:     validatorAccount.Ecdsa.Address(),
+		Input:    encoded,
+		To:       (*ethgo.Address)(&contracts.HydraChainContract),
+		GasPrice: sidechainHelper.DefaultGasPrice,
```

Comment:
better not to pass gasPrice so it is calculated based on the node info about the current gas price